src/hardware/mpu401.cpp

```diff
@@ -150,16 +158,16 @@ static void MPU401_WriteCommand(io_port_t, const io_val_t value, io_width_t)
 		// MIDI stop, start, continue
 		switch (val & 3) {
 		case 1:
-			MIDI_RawOutByte(0xfc);
+			MIDI_RawOutByte(MidiStatus::Stop);
 			mpu.clock.cth_savecount = mpu.clock.cth_counter;
 			break;
 
 		case 2:
-			MIDI_RawOutByte(0xfa);
+			MIDI_RawOutByte(MidiStatus::Start);
 			mpu.clock.cth_counter = mpu.clock.cth_savecount = 0;
 			break;
 		case 3:
-			MIDI_RawOutByte(0xfb);
+			MIDI_RawOutByte(MidiStatus::Continue);
 			mpu.clock.cth_counter = mpu.clock.cth_savecount;
 			break;
```

Comment:
All of a sudden, the switch statement became self-documenting!

Reply:
Yeah, that's exactly what I'm striving for! 😄